### ISSUE TITLE
Review "weight" of base objects

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#2071]
Currently most our mass/weight settings are probably pretty arbitrary. This can result in *very* different results when throwing objects. A stone block may barely fly while if you try one of the starting tools like a pickaxe it may fly off like it has rockets attached.

We should check the basic items and make sure they have more sensible mass/weight so they won't fly quite as weirdly.

**Comment by tyuiwei** [comment]
I was planning on getting this fixed asap and hope this isn't too much of a dumb question, but I can't seem to find where these are defined. I thought they would be in the prefabs, but...am I wrong..?

**Comment by Cervator** [comment]
@tyuiwei without a lot of looking around (trying to finish up for the evening, late here!) I expect you'll find them in two places: basic block files and prefab *templates* - aka parent prefabs. So all basic tools might inherit from a base item that defines the default mass. I haven't double checked but that's my thought.

@SkySom is having all kinds of fun with mass while trying to fix Rails so he might also have a clue (he's on IRC a lot, setting fire to things, same on Slack)

**Comment by SkySom** [comment]
Hey now. I figured that one out. Had nothing to do with the mass! had to do with the fact that Prefabs don't like having comments in them. (There was something commented out, which I had yet to touch before then.) When I finally removed the comments, everything worked. And I totally didn't set anything on fire in slack.

But on to the actual question: Mass is defined in a RigidBody Component as shown here: [RigidBody](https://github.com/MovingBlocks/Terasology/blob/21a20d2c043afe129b0b28dd6d355f8dcdc45559/engine/src/main/resources/assets/prefabs/itemPickup.prefab#L10) From what I've come to find out. This is sorta what affects that. Though someone with a better understanding of the actual physics engine would probably be better suited then I.
